### DISCUSSION TITLE
Full Stack Developer

### DISCUSSION THREAD

**Discussion by dbltw** [community/community#127710] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi, we are searching for a Full Stack Developer with experience in SAP Commerce for a company based in Madrid (La Torre Wallace). Requirements: knowledge of Java, Spring, HTML, CSS, and JavaScript. Tasks: customization of SAP Commerce and content management. Remote work is possible. If you are interested, please get in touch.
Thank you.

**Comment by ebndev** [comment]
Hi @dbltw, thanks for being a part of the GitHub Community!

Please note: we currently do not allow job listings on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
